Bug: demo.launch.py fails with missing control_gains['hand'] for arm_type:=v10

### Describe the problem you got

**Description:**
Launching the bimanual MoveIt demo with real hardware fails during URDF/Xacro processing because the generated robot description expects a hand entry inside control_gains.yaml, but the YAML file does not contain that key.

There also appears to be a confusing version naming mismatch between:
openarm_v1.0 asset directory
arm_type:=v10 launch argument

The launch currently emits a warning/error suggesting only v10 is supported even when using the openarm_v1.0 assets.

**Environment**
Ubuntu 24.04
ROS 2 Jazzy
Hardware: OpenArm dual-arm setup
CAN-FD configured on:
can0
can1

**Repositories:**
openarm_ros2
openarm_description
openarm_bimanual_moveit_config

Installed from latest main branch on GitHub.

**Command Used**
ros2 launch openarm_bimanual_moveit_config demo.launch.py \
  arm_type:=v10 \
  use_fake_hardware:=false \
  right_can_interface:=can0 \
  left_can_interface:=can1
Actual Behavior

**Launch fails with:**
assets/robot/openarm_v1.0 only supports arm_type:=v10. Use assets/robot/openarm_v2.0 for newer hardware variants.
when processing file:
/root/ros2_ws/install/openarm_description/share/openarm_description/assets/robot/openarm_v1.0/urdf/openarm_v10.urdf.xacro
[ERROR] [launch]:
Caught exception in launch (see debug for traceback):
The yaml dictionary has no key 'hand' when evaluating expression 'control_gains['hand']['kp']'

--- a/assets/robot/openarm_v1.0/config/arm/control_gains.yaml
+++ b/assets/robot/openarm_v1.0/config/arm/control_gains.yaml
@@ -39,3 +39,7 @@ joint6:
 joint7:
   kp: 10.0
   kd: 0.5
+
+hand:
+  kp: 5.0
+  kd: 0.1

